planning/behavior_path_planner/src/utils/goal_planner/default_fixed_goal_planner.cpp

```diff
@@ -38,12 +40,49 @@ BehaviorModuleOutput DefaultFixedGoalPlanner::plan(
   return output;
 }
 
+lanelet::ConstLanelets DefaultFixedGoalPlanner::extractLaneletsFromPath(
+  const PathWithLaneId & refined_path,
+  const std::shared_ptr<const PlannerData> & planner_data) const
+{
+  const auto & rh = planner_data->route_handler;
+  lanelet::ConstLanelets refined_path_lanelets;
+  for (size_t i = 0; i < refined_path.points.size(); ++i) {
+    const auto & path_point = refined_path.points.at(i);
+    int64_t lane_id = path_point.lane_ids.at(0);
+    lanelet::ConstLanelet lanelet = rh->getLaneletsFromId(lane_id);
+    bool is_unique = true;
+    for (const lanelet::ConstLanelet & existing_lanelet : refined_path_lanelets) {
+      if (lanelet == existing_lanelet) {
+        is_unique = false;
+        break;
+      }
+    }
+    if (is_unique) {
```

Comment:
can be replaced with `std::find`

Reply:
replaced by `std::find`
[https://github.com/autowarefoundation/autoware.universe/commit/d7509b717ac178d760b31d758201fa9d4865e8f3](https://github.com/autowarefoundation/autoware.universe/commit/d7509b717ac178d760b31d758201fa9d4865e8f3)